src/views/v2/commerce/order/attendees.php

```diff
@@ -13,26 +13,33 @@
  *
  * @version TBD
  *
- * @var \Tribe__Template $this                  [Global] Template object.
- * @var Module           $provider              [Global] The tickets provider instance.
- * @var string           $provider_id           [Global] The tickets provider class name.
- * @var \WP_Post         $order                 [Global] The order object.
- * @var int              $order_id              [Global] The order ID.
- * @var bool             $is_tec_active         [Global] Whether `The Events Calendar` is active or not.
- * @var array            $attendees             [Global] List of attendees for the given order.
+ * @var \Tribe__Template $this          [Global] Template object.
+ * @var Module           $provider      [Global] The tickets provider instance.
+ * @var string           $provider_id   [Global] The tickets provider class name.
+ * @var \WP_Post         $order         [Global] The order object.
+ * @var int              $order_id      [Global] The order ID.
+ * @var bool             $is_tec_active [Global] Whether `The Events Calendar` is active or not.
+ * @var array            $attendees     [Global] List of attendees for the given order.
  */
 
 if ( empty( $order ) || empty( $attendees ) ) {
 	return;
 }
 
+$parent_classes = [
```

Comment:
Let's go with `wrapper_classes` so that we're consistent with what we're doing in other templates.